backend/api/Controllers/ReturnToHomeController.cs

```diff
@@ -35,19 +35,19 @@ [FromRoute] string robotId
                 return NotFound();
             }
 
-            var returnToHomeMission =
-                await returnToHomeService.ScheduleReturnToHomeMissionRunIfNotAlreadyScheduled(
-                    robot,
-                    true
-                );
-            if (returnToHomeMission is null)
+            try
             {
-                string errorMessage = "Error while scheduling Return Home mission";
-                logger.LogError(errorMessage);
-                return StatusCode(StatusCodes.Status502BadGateway, $"{errorMessage}");
+                await isarService.ReturnHome(robot);
```

Comment:
This functions looks like it can throw more exceptions then you are catching, is this intentional?

Reply:
That is unintended. Added a general exception catch